src/sentry/static/sentry/less/sentry.less

```diff
@@ -45,3 +45,4 @@
 @import url("./setup-wizard.less");
 @import url("./spacing.less");
 @import url("./stream.less");
+@import url("./QueryCount.less");
```

Comment:
This is already merged, but can we keep to the same camelCase naming scheme we're rolling going forward? (Until such a time as we changed it globally.)